articles/azure-monitor/app/sdk-connection-string.md

```diff
@@ -19,7 +19,7 @@ Key-value pairs provide an easy way for users to define a prefix suffix combinat
 
 [!INCLUDE [azure-monitor-log-analytics-rebrand](../../../includes/azure-monitor-instrumentation-key-deprecation.md)]
 
-Don't use a connection string and instrumentation key simultaneously. Whichever was set last will take precedence.
+While both the Instrumentation Key and Connection String can be set in an Azure Functions application, the Connection String takes precedence in determining the connection to ApplicationInsights and the Instrumentaion Key is ignored.
```

Comment:
```suggestion
 
```

@RohitRanjanMS Thanks for the feedback. I'm requesting that we delete line  22 for the following reasons: 
- Mentioning anything about using both (“Whichever was set last…”) seems contradictory to the “transition” messaging in the Note directly above the text being edited by the PR.
![image](https://github.com/MicrosoftDocs/azure-docs/assets/108769795/5ce90a89-0aa8-4595-9721-756d687a9743)
- In the [Troubleshooting](https://learn.microsoft.com/en-us/azure/azure-monitor/app/migrate-from-instrumentation-keys-to-connection-strings#troubleshooting) section for Migrate from Application Insights instrumentation keys to connection strings, it sounds like using both could lead to data being lost.
![image](https://github.com/MicrosoftDocs/azure-docs/assets/108769795/0caa8b02-eea5-445c-9978-d79da6e649b2)